node-control.sh

```diff
@@ -8,6 +8,29 @@ function print_help {
   echo "Usage: $0 {attach|detach} <node-number>"
 }
 
+function wait_for_pg {
+    set +e
+    PG_HOST=$1
+
+    PSQL_UP=0
+    for i in {0..60}; do
+        pg_isready -h $PG_HOST
+        if [ $? -eq 0 ]; then
+            PSQL_UP=1
+            break
+        fi
+        sleep 15
+    done
+
+    if [ $PSQL_UP -ne 1 ]; then
+        echo "PSQL did not start within time. Aborting."
+        exit -1
```

Comment:
in bash, exit codes are positive numbers

Reply:
Fixed.